q2_vsearch/assets/fastq_eestats2.html

```diff
@@ -0,0 +1,24 @@
+{% extends "tab-child.html" %}
+
+{% block content %}
+<div class="row">
+    <div class="col-lg-12">
+        <h1>vsearch - fastq_eestats2</h1>
+        {% set colsize = 6 if paired else 12 %}
+        <div class="col-lg-{{ colsize }}">
```

Comment:
Missing a parent `<div class="row">` above, which is causing bootstrap to render a little strangely in the viz.

Reply:
Sorry, I don’t get it. I compared the `html` with other visualizations. Every row has its `<div class=”row”>` or do you mean to get the headline an own `<div>`?

Reply:
The parent element for any columns should be a row (in bootstrap). Here, your col is nested directly inside another col.